### ISSUE TITLE
Broken indentation with each expression

### ISSUE THREAD

**Issue by NGPixel** [negibouze/html-webpack-pug-plugin#23]
Using `html-webpack-pug-plugin` version `1.0.1`, the indentation below `each` expressions is lost in the transformed result, e.g.:

```pug
html
  head
    each favsize in [57, 60, 72, 76, 114, 120, 144, 152, 180]
      link(rel='apple-touch-icon', sizes=favsize + 'x' + favsize, href='/favicons/apple-icon-' + favsize + 'x' + favsize + '.png')
    link(rel='icon', type='image/png', sizes='192x192', href='/favicons/android-icon-192x192.png')
    each favsize in [32, 96, 16]
      link(rel='icon', type='image/png', sizes=favsize + 'x' + favsize, href='/favicons/favicon-' + favsize + 'x' + favsize + '.png')
    link(rel='manifest', href='/manifest.json')
```
is incorrectly transformed to
```pug
html
  head
    each favsize in [57, 60, 72, 76, 114, 120, 144, 152, 180]
    link(rel='apple-touch-icon', sizes=favsize + 'x' + favsize, href='/favicons/apple-icon-' + favsize + 'x' + favsize + '.png')
    link(rel='icon', type='image/png', sizes='192x192', href='/favicons/android-icon-192x192.png')
    each favsize in [32, 96, 16]
    link(rel='icon', type='image/png', sizes=favsize + 'x' + favsize, href='/favicons/favicon-' + favsize + 'x' + favsize + '.png')
    link(rel='manifest', href='/manifest.json')
```

The indentation should be retained in the final result.

Thanks!

**Comment by negibouze** [comment]
@NGPixel Thank you for your report.

**Comment by negibouze** [comment]
@NGPixel 
Sorry for my late reply.
I released a new version(v2.0.0).
Please try it.

**Comment by NGPixel** [comment]
👍 Thanks that works
